ui/src/pub.vue

```diff
@@ -385,10 +358,20 @@ export default {
 
             query: "",
             config: Vue.config,
+
+            tab: 0,
+            tabs: [
```

Comment:
Yesterday we decided not to use tab after all. I don't think we need to use tabs for pub page because it makes it difficult for visitor to find information if it's hidden under different tab.